db/schema.rb

```diff
@@ -72,6 +100,59 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "question_audios", force: :cascade do |t|
+    t.integer "question_text_id"
+    t.integer "size", null: false
+    t.integer "length", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["question_text_id"], name: "index_question_audios_on_question_text_id"
+  end
+
+  create_table "question_inputs", force: :cascade do |t|
+    t.integer "question_id"
+    t.integer "allow_multiple_selections"
+    t.integer "text_numeric"
+    t.integer "text_alphanumeric"
+    t.integer "audio"
+    t.text "default_value"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["question_id"], name: "index_question_inputs_on_question_id"
+  end
+
+  create_table "question_question_texts", force: :cascade do |t|
+    t.integer "question_id"
+    t.integer "question_text_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["question_id"], name: "index_question_question_texts_on_question_id"
+    t.index ["question_text_id"], name: "index_question_question_texts_on_question_text_id"
+  end
+
+  create_table "question_texts", force: :cascade do |t|
+    t.integer "language_id"
+    t.integer "text", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["language_id"], name: "index_question_texts_on_language_id"
+  end
+
+  create_table "question_types", force: :cascade do |t|
+    t.string "type_name", null: false
```

Comment:
Let's simplify the column name to `name`. It's implied from the table name that it's a type of some kind.